mmdet/apis/ote/apis/detection/openvino_task.py

```diff
@@ -182,3 +210,63 @@ def evaluate(self,
                  output_result_set: ResultSet,
                  evaluation_metric: Optional[str] = None):
         return MetricsHelper.compute_f_measure(output_result_set).get_performance()
+
+    def optimize(self,
+                 optimization_type: OptimizationType,
+                 dataset: Optional[Dataset],
+                 output_model: OptimizedModel,
+                 optimization_parameters: Optional[OptimizationParameters]):
```

Comment:
It might be useful to update optimization progress via `optimization_parameters.update_progress` if it is possible.

Reply:
Will try it in next PR